source/common/stream_info/stream_info_impl.h

```diff
@@ -174,12 +174,20 @@ struct StreamInfoImpl : public StreamInfo {
 
   absl::optional<uint32_t> responseCode() const override { return response_code_; }
 
+  absl::optional<Grpc::Status::GrpcStatus> localReplyGrpcStatus() const override {
```

Comment:
Since this is specific for "localReply",  are we only allowed it to be called by sendLocalReply()?